webapp/src/components/RadioField/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {ReactNode} from 'react';
+
+import {RadioOptionsType} from 'types';
+
+type Props = {
+    heading: string | ReactNode;
+    isInline: boolean;
+    options: RadioOptionsType[];
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    description: string | ReactNode;
+}
+
+export const RadioField = ({heading, isInline, options, onChange, description}: Props) => {
+    return (
+        <div className='form-group'>
+            <label className='col-sm-4'>
+                {heading}
+            </label>
+            <div className='col-sm-8'>
+                <div className={`${!isInline && 'radio'}`}>
```

Comment:
If `isInline` is `true`, we will be setting `className` to `false` which is probably not what we want to do here